agent/src/beerocks/slave/backhaul_manager/backhaul_manager_thread.cpp

```diff
@@ -120,6 +120,16 @@ bool backhaul_manager::init()
         return false;
     }
 
+    if (m_sConfig.ucc_listener_port != 0) {
```

Comment:
We should add a sanity test which verifies the selected port is a valid port which is not used by another application (nitpick)

Reply:
I think there is, and it is done by the socket thread. The bind function will fail on socket thread if someone using it.